src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,137 @@
+@import '../utils/variables';
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: #{$card-width};
+  height: #{$card-height};
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: rgba(255, 255, 255, 1);
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  transition: transform 0.3s ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  .card__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: $bottom-margin-image;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
```

Comment:
no need too repeat card everywhere, just use `&` where you have nesting elements  ->`&__img`